README.md

```diff
@@ -131,7 +146,28 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD shows how all records in a database relate,
+and what attributes they have. That is: how the tables in a database refer to
+each other, and what columns those tables may have.
+
+Genie
+  - wishes_left
+  - color
+  - has_one :lamp
+Lamp
```

Comment:
Good answer! Not really a right or wrong here but it is really similar to the solution answer though